### PR TITLE
feat: display user and ticket information

### PR DESCRIPTION
**Pull request by TonNQ**
Display user and ticket information after payment successfully

86epq42ae

### CODE REVIEW

File: src/components/MovieShows/MovieShows.jsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+      </div>
+      <div className='flex w-full flex-col'>
+        <div className='flex w-full flex-col'>
+          <h2 className='text-xl font-bold uppercase'>{movie.infor.filmName}</h2>
+          <div className='mt-2 flex flex-wrap items-center gap-x-2 gap-y-2'>
+            {MOVIE_INFO(movie.infor).map(({ text, icon }) => {
+              if (text) {
+                const IconComponent = icon
+                return (
+                  <div className='flex items-center gap-1 text-sm font-semibold' key={icon}>
```
Comment (ces-phatnguyen) [inline src/components/MovieShows/MovieShows.jsx:30]: key should be number or string only

Comment (TonNQ) [reply src/components/MovieShows/MovieShows.jsx:30]: Can I use index (as number) for key?

**Review by ces-khanhhoang — APPROVED** [review]

File: src/components/MovieShows/MovieShows.jsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+          src={imageError ? ImageEmpty : movie.infor.poster}
+          onError={handleLoadImageError}
+          className='object-container h-auto w-full rounded-md'
+        />
+      </div>
+      <div className='flex w-full flex-col'>
+        <div className='flex w-full flex-col'>
+          <h2 className='text-xl font-bold uppercase'>{movie.infor.filmName}</h2>
+          <div className='mt-2 flex flex-wrap items-center gap-x-2 gap-y-2'>
+            {MOVIE_INFO(movie.infor).map(({ text, icon }, index) => {
```
Comment (ces-anhlam) [inline src/components/MovieShows/MovieShows.jsx:26]: `{ text, icon: IconComponent}`

File: src/components/MovieShows/MovieShows.jsx [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+              if (text) {
+                const IconComponent = icon
+                return (
+                  <div className='flex items-center gap-1 text-sm font-semibold' key={index}>
```
Comment (ces-anhlam) [inline src/components/MovieShows/MovieShows.jsx:30]: `{text && <div ....}`

File: src/components/MovieShows/MovieShows.jsx [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+                    <span>{text}</span>
+                  </div>
+                )
+              }
+            })}
+          </div>
+        </div>
+        <div className='mt-4 flex w-full flex-col gap-y-[6px]'>
+          {movie.showTime.map((showTime) => (
+            <ShowsWithinDay key={showTime} showTime={showTime} />
```
Comment (ces-anhlam) [inline src/components/MovieShows/MovieShows.jsx:41]: Do you have any ID in `showTime`. Don't put object to key

File: src/components/UserInfoContainer/UserInfoContainer.jsx [hunk 5]
```diff
@@ -1,3 +1,30 @@
...
+  const handleGoBackHome = () => {
+    window.location.href = `${import.meta.env.VITE_WEB_URL}${path.home}`
+  }
+
+  if (!_.isEmpty(userInfo)) {
+    return (
+      <>
+        <div className='flex w-full flex-col gap-4 rounded-lg bg-gradient-to-tl from-blue-custom-700 to-purple-custom-700 p-8 text-white-custom-700'>
+          <h1 className='w-full text-center text-3xl font-bold uppercase'>User information</h1>
+          {USER_INFO(userInfo).map((info) => (
```
Comment (ces-anhlam) [inline src/components/UserInfoContainer/UserInfoContainer.jsx:19]: `map(({label, value})) => ...`

File: src/redux/slices/bookTicket.js [hunk 6]
```diff
@@ -118,7 +118,7 @@ export const getTicketInfo = createSelector(
     seatList: seatList,
     seatNames: seatNamesString,
     screenName,
-    totalPrice,
+    totalPrice: totalPrice,
```
Comment (ces-anhlam) [inline src/redux/slices/bookTicket.js:121]: `totalPrice` is enough

File: src/components/MovieShows/MovieShows.jsx (hunk 2 above)
Comment (TonNQ) [reply src/components/MovieShows/MovieShows.jsx:26]: fixed

File: src/components/MovieShows/MovieShows.jsx (hunk 3 above)
Comment (TonNQ) [reply src/components/MovieShows/MovieShows.jsx:30]: fixed

File: src/components/UserInfoContainer/UserInfoContainer.jsx (hunk 5 above)
Comment (TonNQ) [reply src/components/UserInfoContainer/UserInfoContainer.jsx:19]: fixed

File: src/redux/slices/bookTicket.js (hunk 6 above)
Comment (TonNQ) [reply src/redux/slices/bookTicket.js:121]: fixed

File: src/components/MovieShows/MovieShows.jsx (hunk 4 above)
Comment (TonNQ) [reply src/components/MovieShows/MovieShows.jsx:41]: fixed

**Review by ces-anhlam — APPROVED** [review]

**Pull request merged**